src/launcher/index.css

```diff
@@ -0,0 +1,137 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+
+.jp-Launcher {
+  position: absolute;
```

Comment:
The `position`, `top`, `left`, `width` and `height` properties should not be set. They are managed automatically by the Phosphor layout. The `z-index` is also unnecessary.